ui/espresso/BasicSample/BUILD.bazel

```diff
@@ -21,7 +20,7 @@ android_library(
 android_binary(
```

Comment:
Off topic, can you add license/copyright to these bazel files

Reply:
Done, added license header.